frontier-chain/precompile/living-assets/contracts/LivingAssetsEvolution.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Pallet Collection Manager Interface
+/// @author Freeverse team
+/// @notice This interface allows Solidity contracts to interact with pallet-living-assets
+/// @custom:address 0x0000000000000000000000000000000000000402
+interface LivingAssets {
+    /// @notice Event emitted when a new collection is created
+    /// @param collectionId Id of the newly created ERC721 collection
+    /// @param owner Owner of the newly created ERC721 collection
+    event CreateCollection(uint256 collectionId, address indexed owner);
```

Comment:
I think that a standard solidity way of naming events is that it should be called "NewCollection" instead of "CreateCollection". I would suggest NewCollection.

Reply:
I don't have a favored term for it, yet I couldn't find any standard terminology in Solidity regarding this. Could you provide a link on the topic?